server/controllers/finance/priceList.js

```diff
@@ -229,6 +236,92 @@ exports.createItem = function createItem(req, res, next) {
     .done();
 };
 
+
+/**
+ * @method downloadTemplate
+ *
+ * @description send to the client the template file for price list item import
+*/
+exports.downloadTemplate = (req, res, next) => {
+  try {
+    const file = path.join(__dirname, '../../resources/templates/import-inventory-item-template.csv');
+    res.download(file);
+  } catch (error) {
+    next(error);
+  }
+};
+
+exports.downloadFilledTemplate = async (req, res, next) => {
+  try {
+    const pdfOptions = {
```

Comment:
This is only available as CSV, right?  Or can we download it in other formats?

If only CSV, you can probably remove most of this code (for example, `orientation` doesn't apply to CSVs, just PDFs).